Makefile

```diff
@@ -170,3 +172,35 @@ stretch: 17.12.0-stretch 17.09.1-stretch 17.09.0-stretch 17.06.2-stretch 17.03.2
                         --build-arg PKG_REV="$(PKG_REV)" \
                         -t nvidia-docker2:$@ -f Dockerfile.stretch .
 	$(DOCKER) run --rm -v $(DIST_DIR)/stretch:/dist:Z nvidia-docker2:$@
+
+17.12.0.ce-amzn1:
```

Comment:
FYI you could combine these into one rule.

```makefile
17.12.0.ce-amzn1 17.09.1.ce-amzn1 17.06.2.ce-amzn1 17.03.2.ce-amzn1:
	$(DOCKER) build --build-arg RUNTIME_VERSION="$(RUNTIME_VERSION)-1.docker$(subst .ce-amzn1,,$@)" \
                        --build-arg DOCKER_VERSION="docker = $(subst -amzn1,,$@)" \
                        --build-arg PKG_VERS="$(VERSION)" \
                        --build-arg PKG_REV="$(PKG_REV).docker$(subst -amzn1,,$@)" \
                        -t nvidia-docker2:$@ -f Dockerfile.amzn1 .
	$(DOCKER) run --rm -v $(DIST_DIR)/amzn1:/dist:Z nvidia-docker2:$@
```

Reply:
Not a big fan of this, but we could do that since it's getting a bit out of control.

Reply:
It was merely a suggestion, but I'm just curious of what _specifically_ do you find unappealing?